Automattic/wp-calypso
# Site monitoring: reduce download logs batch size and modify error notices

- Closes https://github.com/Automattic/dotcom-forge/issues/3834

## Proposed Changes

* Reduce `page_size` to minimize the chances of getting an error retrieving the logs
* Change the error notices when downloading the logs

## Testing Instructions



Happy path:

* On an atomic site, navigate to `/site-monitoring/${siteSlug}/web`
* Select a wide time range to download thousands of entries.
* Click on `Download logs`
* Observe the counter increments by 500 and you download successfully.
* Click on `Error logs` and repeat the test steps.

Error messages:

* Apply this diff to your sandbox and point the public api to your machine
* Repeat the tests on the happy path
* Observe the following error messages for these error codes:
   * `500` : `There was an error retrieving logs. Please try again in a few minutes.`
   * `400`: `There was an error retrieving logs. Please try with a different time range.`


## Screenshots

**Success, using batches of five hundred entries**


https://github.com/Automattic/wp-calypso/assets/779993/264bab68-01df-4cd1-b304-6ab5cab30673


**Error 400**

<img width="1398" alt="Screenshot 2023-09-25 at 15 40 29" src="https://github.com/Automattic/wp-calypso/assets/779993/5ea0ff89-f586-4505-aadf-2123dd7f2c19">


**Error 500**

<img width="1398" alt="Screenshot 2023-09-25 at 15 41 57" src="https://github.com/Automattic/wp-calypso/assets/779993/ad26dab7-ab37-42b6-b926-8ddd57a19d30">


## Pre-merge Checklist
